src/Elemental/View/Form/Field/Select.elm

```diff
@@ -0,0 +1,246 @@
+module Elemental.View.Form.Field.Select exposing
```

Comment:
We should add an examples directory to this project to be able to demo the various views. This way we can see and test them outside of any given project. What do you think?